Update advanced-tutorial.md and other-tutorials.md (#466)
smartcontractkit/documentation: docs/Introduction/tutorials/other-tutorials.md

layout: nodes.liquid
section: smartContract
date: Last Modified
title: "Other Resources"
permalink: "docs/other-tutorials/"
---

> Welcome to the Resources page. We've compiled a list of helpful links and pages you might need to help you out throughout your learning journey. If you're new to Chainlink, we suggest starting with [The Basics](../beginners-tutorial/) tutorial and working through the rest of our tutorial series to better understand the products and services Chainlink offers. This page contains more resources, inspiration, and outreach information to further your learning.

Reviewer: ```suggestion
Welcome to the Resources page. This is list of links and pages you might need to help you out throughout your learning journey. If you're new to Chainlink, start with [The Basics](../beginners-tutorial/) tutorial and work through the rest of the tutorial series to better understand the products and services Chainlink offers. This page contains more resources, inspiration, and outreach information to further your learning.
```

**Table of Contents**

+ [Supplementary Materials](#supplementary-materials)
  + [Video Tutorials](#video-tutorials)
  + [Application Projects](#application-projects)
  + [Starter Kits](#starter-kits)
  + [More Inspiration](#more-inspiration)
+ [Outreach](#outreach)
  + [Get Support](#get-support)
  + [Join the Community](#join-the-community)

# Supplementary Materials

## Video Tutorials

We're constantly uploading new videos to our [YouTube](https://www.youtube.com/c/chainlink) channel. Here are some direct links to playlists that may be helpful if you're starting your Chainlink learning journey or if you're looking for interesting ways to apply Chainlink to your own project:

- [Two-Minute Explainers](https://www.youtube.com/playlist?list=PLVP9aGDn-X0Q3qBME3T9sBMw66xPsglMA)
- [Chainlink Developer Workshops](https://www.youtube.com/playlist?list=PLVP9aGDn-X0SPHromvpiGvoNDpH7YErmf)
- [Chainlink Engineering Tutorials](https://www.youtube.com/playlist?list=PLVP9aGDn-X0QwJVbQvuKr-zrh2_DV5M6J)

## Application Projects

> Below is a list of applications of Chainlink's products and services. We've sorted them by difficulty and type to help you navigate and search for projects you may find interesting. We're always posting about new ways to use Chainlink so make to visit our [blog](https://blog.chain.link/) to stay up to date with our latest posts.

Reviewer: ```suggestion
Below is a list of applications of Chainlink's products and services sorted by difficulty and type to help you navigate and search for projects that you might find interesting. See the [Chainlink Blog](https://blog.chain.link/) to stay up to date with new ways to use Chainlink.
```

layout: nodes.liquid
section: smartContract
date: Last Modified
title: "Learning Resources"
permalink: "docs/other-tutorials/"
---

> Welcome to the Resources page. This is list of links and pages you might need to help you out throughout your learning journey. If you're new to Chainlink, start with [The Basics](../beginners-tutorial/) tutorial and work through the rest of the tutorial series to better understand the products and services Chainlink offers. This page contains more resources, inspiration, and outreach information to further your learning.

Reviewer: ```suggestion
Welcome to the Resources page. This is a list of links and pages that you might need to help you throughout your learning journey. If you're new to Chainlink, start with [The Basics](../beginners-tutorial/) tutorial and work through the rest of the tutorial series to better understand the products and services Chainlink offers. This page contains more resources, inspiration, and outreach information to further your learning.
```

**Table of Contents**

> + [# Smart Contract Learning Materials](#smart-contract-learning-materials)

Reviewer: ```suggestion
+ [Smart Contract Learning Materials](#smart-contract-learning-materials)
```

  + [Video Tutorials](#video-tutorials)
  + [Application Projects](#application-projects)
  + [Game Resources](#game-resources)
  + [Coding Bootcamps](#coding-bootcamps)
  + [Starter Kits](#starter-kits)
  + [External Tutorials](#external-tutorials)
  + [More Inspiration](#more-inspiration)
+ [Outreach](#outreach)
  + [Get Support](#get-support)
  + [Join the Community](#join-the-community)

# Smart Contract Learning Materials

## Video Tutorials

> We're constantly uploading new videos to our [YouTube](https://www.youtube.com/c/chainlink) channel. Here are some direct links to playlists that may be helpful if you're starting your Chainlink learning journey or if you're looking for interesting ways to apply Chainlink to your own project:

Reviewer: ```suggestion
We're constantly uploading new videos to our [YouTube](https://www.youtube.com/c/chainlink) channel. Here are some direct links to playlists that may be helpful if you're starting your Chainlink learning journey or if you're looking for interesting ways to apply Chainlink to your project:
```

| [Build an RFID Blockchain Integration With Chainlink External Adapters](https://blog.chain.link/rfid-blockchain-integration-with-chainlink-external-adapters/) | Identification       | Advanced   |
| [How to Connect a Tesla Vehicle API to a Smart Contract Via a Chainlink External Adapter](https://blog.chain.link/create-tesla-smart-contract-rental/)         | Transportation       | Advanced   |
| [OAuth and API Authentication in Smart Contracts](https://blog.chain.link/oauth-and-api-authentication-in-smart-contracts-2/)                                                                                                                                                               |  Security                    |  Advanced          |

## Starter Kits

> Our Starter Kits help jumpstart your full stack development process. Currently, we have starter kits for Brownie (Python), Truffle (Javascript), and Hardhat (Javascript). Be sure to visit our [Engineering Tutorials](https://www.youtube.com/playlist?list=PLVP9aGDn-X0QwJVbQvuKr-zrh2_DV5M6J) playlist to learn more about each kit and how to use it. 

Reviewer: ```suggestion
Our Starter Kits help jumpstart your full stack development process. You can get starter kits for Brownie (Python), Truffle (Javascript), and Hardhat (Javascript). See the [Engineering Tutorials](https://www.youtube.com/playlist?list=PLVP9aGDn-X0QwJVbQvuKr-zrh2_DV5M6J) playlist to learn more about each kit and how to use it. 
```

- [Chainlink Brownie Starter Kit (Python)](https://github.com/smartcontractkit/chainlink-mix)
- [Chainlink Truffle Starter Kit (Javascript)](https://github.com/smartcontractkit/truffle-starter-kit)
- [Chainlink Hardhat Starter Kit (Javascript)](https://github.com/smartcontractkit/hardhat-starter-kit)

## More Inspiration

> Looking for more ways to use Chainlink? Here are some resources that discuss Chainlink's use cases as well as the winning projects we've had in our previous hackathons. We've also added a link to our Marketplace where you can find our latest data provider nodes and explore our network. 

Reviewer: ```suggestion
Looking for more ways to use Chainlink? Here are some resources that discuss Chainlink's use cases as well as the winning projects we've had in our previous hackathons. There is also link to the Marketplace where you can find the latest data provider nodes and explore the network. 
```

- [77 Use Cases by Chainlink](https://blog.chain.link/44-ways-to-enhance-your-smart-contract-with-chainlink/)
- [Past Hackathon Winning Projects](/docs/example-projects)
- [Chainlink Marketplace](https://market.link/)

# Outreach

> The Chainlink community is some of the most inviting groups of engineers. We are always looking to help users expand their knowledge on Chainlink and solve related issues. Always refer back to the [Getting Help](/docs/getting-help) page for the latest information about how to get support.

Reviewer: ```suggestion
The Chainlink community is an inviting group of engineers that is always looking to help users expand their knowledge on Chainlink and solve related issues. Refer back to the [Getting Help](/docs/getting-help) page for the latest information about how to get support.
```

| [Build an RFID Blockchain Integration With Chainlink External Adapters](https://blog.chain.link/rfid-blockchain-integration-with-chainlink-external-adapters/) | Identification       | Advanced   |
| [How to Connect a Tesla Vehicle API to a Smart Contract Via a Chainlink External Adapter](https://blog.chain.link/create-tesla-smart-contract-rental/)         | Transportation       | Advanced   |
| [OAuth and API Authentication in Smart Contracts](https://blog.chain.link/oauth-and-api-authentication-in-smart-contracts-2/)                                                                                                                                                               |  Security                    |  Advanced          |

## Game Resources

- [CryptoZombies](https://cryptozombies.io/)
- [DamnVulnerableDefi](https://damnvulnerabledefi.xyz/challenges/8.html)
- [Ethernaut](https://ethernaut.openzeppelin.com/)

## Coding Bootcamps

- [ChainShot](https://www.chainshot.com/courses)
- [Consensys Bootcamp](https://consensys.net/academy/bootcamp/)
- [Ivan On Tech](https://academy.moralis.io/courses/chainlink-101)

## Starter Kits

> Our Starter Kits help jumpstart your full stack development process. You can get starter kits for Brownie (Python), Truffle (Javascript), and Hardhat (Javascript). See the [Engineering Tutorials](https://www.youtube.com/playlist?list=PLVP9aGDn-X0QwJVbQvuKr-zrh2_DV5M6J) playlist to learn more about each kit and how to use them. 

Reviewer: ```suggestion
Our Starter Kits help jumpstart your full-stack development process. You can get starter kits for Brownie (Python), Truffle (Javascript), and Hardhat (Javascript). See the [Engineering Tutorials](https://www.youtube.com/playlist?list=PLVP9aGDn-X0QwJVbQvuKr-zrh2_DV5M6J) playlist to learn more about each kit and how to use them. 
```

- [Chainlink Brownie Starter Kit (Python)](https://github.com/smartcontractkit/chainlink-mix)
- [Chainlink Truffle Starter Kit (Javascript)](https://github.com/smartcontractkit/truffle-starter-kit)
- [Chainlink Hardhat Starter Kit (Javascript)](https://github.com/smartcontractkit/hardhat-starter-kit)

## External Tutorials

- [Ethereum Developers](https://ethereum.org/en/developers/tutorials/)
- [useWeb3](https://www.useweb3.xyz/)

## More Inspiration

> Looking for more ways to use Chainlink? Here are some resources that discuss Chainlink's use cases as well as the winning projects we've had in our previous hackathons. There is also link to the Marketplace where you can find the latest data provider nodes and explore the network. 

Reviewer: ```suggestion
Looking for more ways to use Chainlink? Here are some resources that discuss Chainlink's use cases as well as the winning projects we've had in our previous hackathons. There is also a link to the Marketplace where you can find the latest data provider nodes and explore the network. 
```
